opdracht-2-3/styles.css

```diff
@@ -0,0 +1,56 @@
+* {
+    margin: 0;
+}
+
+body {
+    background-color: #f2f3d9;
+    text-align: center;
+}
+
+.navbar {
+    background-color: #151E3F;
+    padding: 20px;
+    color: #f2f3d9;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.icon {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+}
+
+.shortcuts{
+    list-style-type: none;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    flex-basis: 300px;
+}
+
+.header {
+    margin: 30px 20px;
+    color: #030027;
+}
+
+.red {
+    color: #151E3F;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+section {
+    background-color: #C16E70;
+    border-radius: 5px;
+    margin: 20px 20px;
+    padding: 10px;
+    height: 350px;
+    width: 280px;
+    display: flex;
+    flex-direction: row;
+    align-items: flex-end;
+    flex-basis: auto;
+}
```

Comment:
Al om al, in mijn ogen goed gedaan. In de bijwerkingen stonden dingen anders vermeldt maar dat maakt in deze niet uit. Je hebt hetzelfde bereikt dus gewoon goed :-)